packages/config/config/devices/0x021f/dmdp1.json

```diff
@@ -13,7 +13,28 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Set",
+			"description": "Send a Basic Set command when the contact opens or closes.",
```

Comment:
@AlCalzone We don't do descriptions for associtation groups normally? None of the UIs use them anyway to my knowledge.

Reply:
If thats the case, I will gladly remove them

Reply:
Done

Reply:
They can exist though (optional): https://zwave-js.github.io/node-zwave-js/#/config-files/file-format?id=associations